### PR TITLE
PP-92: Fix open issues

### PR DESCRIPTION
**Pull request by antomor**
This is the continuation of [this PR](https://github.com/wilsoniovlabs/relaying-services-sdk-dapp/pull/1):
- Add package-lock.json
- Fix Readme
- Fix other minor issues

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
     "react-scripts": "4.0.3",
-    "relaying-services-sdk": "github:infuy/relaying-services-sdk",
+    "relaying-services-sdk": "https://github.com/wilsoniovlabs/relaying-services-sdk#features/estimate-max-possible-relay-cas-with-linear-fit",
```
Comment (mortelli) [inline package.json]: i've just merged [SDK PR #9](https://github.com/infuy/relaying-services-sdk/pull/9) so we can revert this change :+1:

File: src/App.js [hunk 2]
```diff
@@ -45,7 +45,7 @@ function App() {
                 verbose: window.location.href.includes('verbose')
                 , chainId: process.env.REACT_APP_RIF_RELAY_CHAIN_ID
                 , gasPriceFactorPercent: process.env.REACT_APP_RIF_RELAY_GAS_PRICE_FACTOR_PERCENT
-                , relayLookupWindowBlocks: process.env.REACT_APP_RIF_RELAY_RELAY_LOOKUP_WINDOW_BLOCKS
+                , relayLookupWindowBlocks: process.env.REACT_APP_RIF_RELAY_LOOKUP_WINDOW_BLOCKS
```
Comment (mortelli) [inline src/App.js:48]: it seems the `REACT_APP_RIF_RELAY_RELAY_LOOKUP_WINDOW_BLOCKS` name is still present in the `.env` file

**Review by mortelli — COMMENTED** [review]
some comments

File: src/App.js (hunk 2 above)
Comment (antomor) [reply src/App.js:48]: you're right, thanks for pointing it out.

**Review by mortelli — APPROVED** [review]
